### PR TITLE
Implement proper 3m pattern and update automation

### LINKED ISSUE #97: Use three musketeers approach to reduce pre-requisites for the end user
We have to re-think about how we do the secure site, route53 and how we use three musketeers to reduce the amount of per-requisites. This issue may have to be broken down further.

### PR DESCRIPTION
**Pull request by frdvo**
After 5 months, I got some free time and I Implemented the proper 3m pattern and updated the GH actions automation

You can see the Git Hub Actions working with the proper pattern on my fork  [here](https://github.com/frdvo/2020-jun-project1-group3)

Closes #97 #101 #106

### CODE REVIEW

**Comment by frdvo** [comment]
You can see the Git Hub Actions working with the proper pattern on my fork at [](https://github.com/frdvo/2020-jun-project1-group3)

File: .env [hunk 1]
```diff
@@ -0,0 +1,8 @@
+acm_cert_arn
+AWS_ACCESS_KEY_ID
+AWS_REGION
+AWS_SECRET_ACCESS_KEY
+domain_name
```
Comment (jay-flood) [inline .env:5]: Not a big deal, but having everything in all-caps would look nice ;-)

Comment (jayanath) [inline .env:5]: Not a big deal at all but having everything in all-caps would be a nice touch ☺️

**Review by jayanath — APPROVED** [review]
Great work! 🚀 🔥 🌟

File: .env (hunk 1 above)
Comment (frdvo) [reply .env:5]: I agree. I have to test it to see if it is case sensitive or not. I will fix and open another PR. Thaks for your review!

**Pull request merged**